src/lib/generateSchemaJson.ts

```
@@ -6,6 +7,10 @@ export function generateSchemaJson(schema: any, useExample = false) {
     return '{}'
   }
 
+  if (useTheme().getSpecConfig().avoidCirculars.value) {
+    schema = JSON.parse(formatJson(schema))
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)